vms/platformvm/config/config.go

```diff
@@ -41,32 +43,7 @@ type Config struct {
 	// Set of subnets that this node is validating
 	TrackedSubnets set.Set[ids.ID]
 
-	// Fee that is burned by every non-state creating transaction
-	TxFee uint64
-
-	// Fee that must be burned by every state creating transaction before AP3
-	CreateAssetTxFee uint64
-
-	// Fee that must be burned by every subnet creating transaction after AP3
-	CreateSubnetTxFee uint64
-
-	// Fee that must be burned by every transform subnet transaction
-	TransformSubnetTxFee uint64
-
-	// Fee that must be burned by every blockchain creating transaction after AP3
-	CreateBlockchainTxFee uint64
-
-	// Transaction fee for adding a primary network validator
-	AddPrimaryNetworkValidatorFee uint64
-
-	// Transaction fee for adding a primary network delegator
-	AddPrimaryNetworkDelegatorFee uint64
-
-	// Transaction fee for adding a subnet validator
-	AddSubnetValidatorFee uint64
-
-	// Transaction fee for adding a subnet delegator
-	AddSubnetDelegatorFee uint64
+	fee.StaticConfig
```

Comment:
If we're going to embed this struct I prefer it's at the top of the struct (or we can unembed it as an alternative)